ui/page/dcrdex/dcrdex_page.go

```diff
@@ -53,7 +53,7 @@ func NewDEXPage(l *load.Load) *DEXPage {
 		materialLoader:     material.Loader(l.Theme.Base),
 	}
 
-	if dp.AssetsManager.DexcInitialized() && dp.AssetsManager.DexClient().IsDEXPasswordSet() {
+	if dp.AssetsManager.DEXCInitialized() && dp.AssetsManager.DexClient().InitializedWithPassword() {
```

Comment:
Should just call `InitializedWithPassword` only.

Reply:
> Should just call InitializedWithPassword only.

Errm the first check enures that `mgr.dexc` is non-nil before we can safely access `DEXClient`